### PR TITLE
[yarn test] add pytorch torchmodel example

### PR DESCRIPTION
**Pull request by piaolaidelangman**

### CODE REVIEW

File: python/orca/example/learn/pytorch/fashion_mnist/fashion_mnist.py [hunk 1]
```diff
@@ -118,8 +118,11 @@ def main():
 
     if args.cluster_mode == "local":
         init_orca_context()
-    elif args.cluster_mode == "yarn":
-        init_orca_context(cluster_mode=args.cluster_mode, cores=4, num_nodes=2)
+    elif args.cluster_mode.startswith("yarn"):
+        if args.cluster_mode == "yarn-client":
+            init_orca_context(cluster_mode=args.cluster_mode, cores=4, num_nodes=2)
```
Comment (qiuxin2012) [inline python/orca/example/learn/pytorch/fashion_mnist/fashion_mnist.py:123]: seems only need one init_orca_context

File: python/orca/example/learn/pytorch/super_resolution/super_resolution.py [hunk 2]
```diff
@@ -57,19 +57,24 @@
...
-    additional = None if not exists("dataset/BSDS300.zip") else "dataset/BSDS300.zip#dataset"
-    init_orca_context(cluster_mode="yarn-client", cores=4, num_nodes=2,
+elif opt.cluster_mode.startswith("yarn"):
+    if opt.cluster_mode == "yarn-client":
+        additional = None if not exists("dataset/BSDS300.zip") else "dataset/BSDS300.zip#dataset"
+        init_orca_context(cluster_mode="yarn-client", cores=4, num_nodes=2,
+                      additional_archive=additional)
+    elif opt.cluster_mode == "yarn-cluster":
+        additional = None if not exists("dataset/BSDS300.zip") else "dataset/BSDS300.zip#dataset"
+        init_orca_context(cluster_mode="yarn-cluster", cores=4, num_nodes=2,
```
Comment (qiuxin2012) [inline python/orca/example/learn/pytorch/super_resolution/super_resolution.py:74]: any different between client and cluster mode?

File: python/orca/example/learn/pytorch/fashion_mnist/fashion_mnist.py (hunk 1 above)
Comment (piaolaidelangman) [reply python/orca/example/learn/pytorch/fashion_mnist/fashion_mnist.py:123]: Will delete if-else

File: python/orca/example/torchmodel/train/mnist/main.py [hunk 3]
```diff
@@ -69,16 +74,17 @@ def main():
     args = parser.parse_args()
 
     torch.manual_seed(args.seed)
-
+    if not exists(args.dir):
+        makedirs(args.dir)
     train_loader = torch.utils.data.DataLoader(
-        datasets.MNIST(args.dir, train=True, download=True,
+        datasets.MNIST(args.dir, train=True, download=False,
```
Comment (qiuxin2012) [inline python/orca/example/torchmodel/train/mnist/main.py:80]: User will fail to run this if he doesn't have mnist data downloaded.

File: python/orca/example/learn/pytorch/super_resolution/super_resolution.py (hunk 2 above)
Comment (piaolaidelangman) [reply python/orca/example/learn/pytorch/super_resolution/super_resolution.py:74]:
```
elif opt.cluster_mode.startswith("yarn"):
    additional = None if not exists("dataset/BSDS300.zip") else "dataset/BSDS300.zip#dataset"
    init_orca_context(opt.cluster_mode="yarn-client", cores=4, num_nodes=2,
                      additional_archive=additional)
 ```

File: python/orca/example/torchmodel/train/mnist/main.py (hunk 3 above)
Comment (piaolaidelangman) [reply python/orca/example/torchmodel/train/mnist/main.py:80]: But jenkins failed if download = True.

**Pull request closed without merging**